docs/configure

```diff
@@ -478,10 +480,11 @@ ditch_maxmsgsize:
 		and domain = '${quote_mysql:$domain}' \
 		and users.maxmsgsize > 0 \
 		and users.domain_id=domains.domain_id}{${value}K}fail} Kb.
-  local_part_suffix = -*
-  local_part_suffix_optional
+#  local_part_suffix = -*
+#  local_part_suffix_optional
```

Comment:
I thought I replied to this, but I can't find that reply.
I think one suffix should be enough by default. We could even leave it commented out, so that the administrator would have to knowingly enable this feature.
I think the plus sign might be more common in this role, but I don't have any data to back this opinion.